.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+### PR Template:
+
+* [ ] Is the agreed upon acceptance criteria fullfilled?
```

Comment:
"Are" rather than "is", since "criteria" is plural.

Also, "fulfilled" not "fullfilled".

Reply:
changed, thx!